CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
+* Updated documentation about testing
```

Comment:
Please remove this.

---

There is no benefit in writing about the changes, that doesn't affect existing developers workflow. New developers would read changed `README.md` file anyway.